src/api/index.js

```diff
@@ -0,0 +1,6 @@
+const giphyAPI = {
+    API_KEY: 'Sa7XeM55KtWebQ4JSASjAax5mDscbodO',
```

Comment:
I would advise you to keep your key on a `.env` file not versioned and provide an `.env.example` to express the usage of the var name.